js/Source/Delegators/Delegator.InputMirror.js

```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
         var targetElement = element.getElement(target.selector);
         if (!targetElement) api.warn('Unable to find element for inputMirror selector: '+target.selector);
         if (targetElement && targetElement != element){
-          targetElement.set(target.property || 'value', element.get('value'));
+          var value = element.get('value');
+          if (api.get('stripHTML')){
+            value = value.replace(/</g, '&#60;').replace(/>/g, '&#62;').replace(/\//g, '&#47;');
+          }
```

Comment:
Given that this doesn't actually strip HTML (that would be `value.stripTags()`) I'd name this and the option "allowHTML". You could also consider adding support for `stripHTML` which would remove all of them (so `<b>foo</b>` becomes `foo`).

Also, generally speaking JS style doesn't use curly braces if it's a one liner:

`if (foo) bar();`

Use braces if it's multi-line or if there's an else that is a multiline:

````
if (foo) bar();
else baz();

if (foo){
  bar();
} else {
  baz();
  biz();
}
```